sdcm/nemesis.py

```diff
@@ -466,6 +467,27 @@ def disrupt_rolling_restart_cluster(self):
             raise UnsupportedNemesis('RollingRestartCluster is supported only for kubernetes backends')
         self.cluster.rollout_restart()
 
+    def disrupt_authenticator_switch_and_restart(self):
+        self._set_current_disruption('AuthenticatorSwitchAndRestart %s' % self.target_node)
+        if not self._is_it_on_kubernetes():
+            raise UnsupportedNemesis('AuthenticatorSwitchAndRestart is supported only for kubernetes backends')
```

Comment:
???

Reply:
I will remove this check, the nemesis works on k8s.